core/src/main/java/org/testcontainers/utility/TestcontainersConfiguration.java

```diff
@@ -105,6 +105,10 @@ public String getSocatContainerImage() {
 
     @Deprecated
     public String getVncRecordedContainerImage() {
+        return getVncDockerImageName().asCanonicalNameString();
+    }
+
+    public DockerImageName getVncDockerImageName() {
```

Comment:
Maybe this is my mistake from solving the merge conflict, but why is `getVncDockerImageName()` needed?

Can't you use `getVncRecordedContainerImage()`? (which I admit, has probably a typo, the name is strange)

Reply:
Done, thanks.